examples/capture_parser/main.rs

```diff
@@ -70,6 +71,22 @@ fn parse(path: &Path, cdclient: &mut Cdclient) -> Res<usize> {
 				dbg!(msg);
 			}
 			packet_count += 1
+		} else if file.name().contains("[53-05-") {
```

Comment:
This being in an else if with the other 53-05- packets below will cause all the other server-sent packets not to parse, constrain this to "53-05-00-00"